src/invoices/InvoiceDetails/Information/Information.js

```diff
@@ -22,6 +22,7 @@ import {
 } from '../../../common/components';
 import BatchGroupValue from '../BatchGroupValue';
 import BillTo from './BillTo';
+import { isCancelled } from '../../../common/utils';
```

Comment:
don't mix imports

Reply:
updated

Reply:
plz look how imports are structured and make the same

Reply:
moved import